api/src/certification/enrolment/domain/usecases/get-candidate-import-sheet-data.js

```diff
@@ -1,23 +1,28 @@
 /**
- * @typedef {import('./index.js').SharedSessionRepository} SharedSessionRepository
+ * @typedef {import('./index.js').SessionRepository} SessionRepository
+ * @typedef {import('./index.js').EnrolledCandidateRepository} EnrolledCandidateRepository
  * @typedef {import('./index.js').CertificationCenterRepository} CertificationCenterRepository
  */
 
 /**
  * @param {Object} params
- * @param {sharedSessionRepository} params.sharedSessionRepository
+ * @param {sessionRepository} params.sessionRepository
+ * @param {enrolledCandidateRepository} params.enrolledCandidateRepository
```

Comment:
Les liens ne fonctionnent pas

```suggestion
 * @param {SessionRepository} params.sessionRepository
 * @param {EnrolledCandidateRepository} params.enrolledCandidateRepository
 * @param {CertificationCenterRepository} params.certificationCenterRepository
```